examples/bzlmod/MODULE.bazel

```diff
@@ -37,6 +37,54 @@ python.toolchain(
     python_version = "3.10",
 )
 
+# One can override the actual toolchain versions that are available, which can be useful
+# to when optimizing what gets downloaded and when.
+python.override(
+    available_python_versions = [
+        "3.10.9",
+        "3.9.19",
+        # The following is used by the `other_module` and we need to include it here
+        # as well.
+        "3.11.8",
+    ],
+    # Also override the `minor_mapping` so that when the modules specify a particular
+    # `3.X` version, we decide what gets used.
```

Comment:
```suggestion
    # Also override the `minor_mapping` so that the root module,
    # instead of rules_python's defaults, controls what full version
    # is used when `3.x` is requested.
```